book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
According to the project requirements, kindly create the `has-many` side of `Book` and `Rental` (a book has many rentals).

![image](https://github.com/J-C-S-V/school-library-ruby/assets/78251927/7b61fb64-d996-4d94-99d7-d47d8f06569f)

Please, add a method for adding rentals, the book itself should be sent as a parameter to create the new rental to achieve the `has-many` association. 😊🛠